doc/neps/nep-0055-string_dtype.rst

```diff
@@ -1,7 +1,7 @@
 .. _NEP55:
 
 =========================================================
-NEP 55 — Add a UTF-8 Variable-Width String DType to NumPy
+NEP 55 — Add a utf-8 variable-width string dtype to NumPy
```

Comment:
Usually dtype has no caps, but it's DType throughouht this NEP. What should be done?

```suggestion
NEP 55 — Add a UTF-8 variable-width string dtype to NumPy
```